spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

```diff
@@ -2357,6 +2358,41 @@ For more details refer to {spring-kafka-docs}#micrometer-native[Micrometer Nativ
 
 
 
+[[production-ready-metrics-mongodb]]
+==== MongoDB Metrics
+Auto-configuration will register a `MongoMetricsCommandListener` for the auto-configured MongoClient.
+
+A timer metric with the name `mongodb.driver.commands` is created for each command issued to the underlying MongoDB driver.
```

Comment:
I would rather have put a line that says "For more information, see the Micrometer docs...." - however there are no MM docs for this fine-grained aspect. Their docs are pretty coarse-grained.